atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -485,10 +483,7 @@ private static Supplier<ExecutorService> instrumentedFixedThreadPoolSupplier(
             int numberOfThriftHosts = serversConfig.numberOfThriftHosts();
             int corePoolSize = poolSize * numberOfThriftHosts;
             int maxPoolSize = maxConnectionBurstSize * numberOfThriftHosts;
-            return Tracers.wrap(MetricRegistries.instrument(
-                    registry,
-                    createThreadPoolWihtoutSpans("Atlas Cassandra KVS", corePoolSize, maxPoolSize),
-                    "Atlas Cassandra KVS"));
+            return createThreadPoolWithoutSpans("Atlas Cassandra KVS", corePoolSize, maxPoolSize);
```

Comment:
Not a problem with this PR, but the existing method naming here is wild to me. `createThreadPoolWithoutSpans` calls through to `com.palantir.common.concurrent.PTExecutors#wrapWithoutSpan` and ends up wrapping with `Tracers#wrap(ExecutorService)` 😱

Reply:
iirc `Tracers.wrap(ExecutorService)` (unlike the overload with a name string) passes along tracing state from the calling thread, but doesn't create a new tracing span for each execution.